client/starwhale/api/_impl/job.py

```diff
@@ -319,3 +402,57 @@ def generate_jobs_yaml(
         ),
         parents=True,
     )
+
+
+class HanderInput(ABC):
+    def __init__(self, required: bool = False) -> None:
+        self.required = required
+
+    @abstractmethod
+    def parse(self, user_input: t.Any) -> t.Any:
+        ...
+
+
+class ListInput(HanderInput):
+    def __init__(self, member_type: t.Any, required: bool = False) -> None:
```

Comment:
List only contains one type?

Reply:
yes, this feature only supports List with same type